pkg/log/log.go

```diff
@@ -29,6 +32,7 @@ func NewLogger(level string, output string) Logger {
 	zerolog.SetGlobalLevel(lvl)
 
 	var log zerolog.Logger
+	var audit_log zerolog.Logger
```

Comment:
don't use underscore for variables, use camelCase.

eg: auditLog

Reply:
I will change that. Thank you, @peusebiu

Reply:
Did the linter not catch this?